Example/TABTestKit/ViewController.swift

```diff
@@ -9,15 +9,59 @@
 //
 
 import UIKit
+import LocalAuthentication
 
 class ViewController: UIViewController {
-  @IBOutlet var helloWorldLabel: UILabel!
-  @IBOutlet var testTextField: UITextField!
+  @IBOutlet private var helloWorldLabel: UILabel!
+  @IBOutlet private var testTextField: UITextField!
+  @IBOutlet private var biometricsStateLabel: UILabel!
+  @IBOutlet private var authenticateButton: UIButton!
   
-    override func viewDidLoad() {
-      helloWorldLabel.text = "Hello world!"
-      testTextField.accessibilityIdentifier = "Test Text Field"
-      helloWorldLabel.accessibilityIdentifier = "Hello World Label"
+  @IBAction private func authenticateButtonTapped(_ sender: UIButton) {
+    let context = LAContext()
+    context.evaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, localizedReason: "Authenticate now!", reply: { _, _ in })
+  }
+  
+  override func viewDidLoad() {
+    configure()
+    setupNotificationListener()
+    setup()
+  }
+  
+  @objc private func setup() {
+    let context = LAContext()
+    var error: NSError?
+    let errorPointer: NSErrorPointer = NSErrorPointer(&error)
+    let isBiometryAvailable = context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: errorPointer)
+    if isBiometryAvailable {
+      handleBiometryAvailable()
+    } else {
+      handleBiometryUnavailable()
     }
+  }
+  
+  private func setupNotificationListener() {
+    NotificationCenter.default.addObserver(self, selector: #selector(setup), name: NSNotification.Name.UIApplicationDidBecomeActive, object: nil)
```

Comment:
I don't think this is needed either sorry, XCUIApplication has a terminate method so you can just kill the app and relaunch in a test with App.shared.terminate() App.shared.launch()

Reply:
I added it because after enrolling the biometrics, I am backgrounding and foregrounding the app to make the status label update.